includes/class-sensei-course-participants.php

```diff
@@ -274,6 +303,28 @@ public function get_course_id() {
 		return $course_id;
 	}
 
+	/**
+	 * Get enrolled participants ids
+	 * 
+	 * @since 2.0.1
+	 *
+	 * @param int  $course_id         Course ID.
+	 * @param bool $exclude_completed Flag if should exclude the completed participants.
+	 * 
+	 * @return void
+	 */
+	private function get_enrolled_participants_ids( $course_id, $exclude_completed ) {
+		$user_ids = Sensei_Course_Enrolment::get_course_instance( $course_id )->get_enrolled_user_ids();
+
+		if ( $exclude_completed ) {
+			$user_ids = array_filter( $user_ids, function( $user_id ) use ($course_id) {
+				return ! WooThemes_Sensei_Utils::user_completed_course( $course_id, $user_id );
```

Comment:
This is a deprecated class name. We can switch to `Sensei_Utils` throughout.

Reply:
Perfect! https://github.com/woocommerce/sensei-course-participants/pull/53/commits/9325c9fcc410827e0d2f874799c3b6328d3b4298